hindi/bollywood.md

```diff
@@ -26,3 +26,5 @@ Art of Living [Know More](art-of-living/art-of-living.md)
 [Sunny Deol](Sunny_Deol/sunny.md)
 
 [चुटकुला पढो](joke/joke.md) और मुस्कुराते रहो
+
+Click [Here](king/king.md) to read the story of a great King.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.